scripts/openmc-make-stopping-powers

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+from urllib.parse import urlencode
+from urllib.request import urlopen
+from lxml import html
+
+import numpy as np
+import h5py
+from openmc.data import ATOMIC_SYMBOL
+
+
+base_url = 'https://physics.nist.gov/cgi-bin/Star/e_table-t.pl'
+energies = np.logspace(-3, 3, 200)
+data = {'matno': '', 'Energies': '\n'.join(str(x) for x in energies)}
+columns = {1: 's_collision', 2: 's_radiative'}
+
+# ==============================================================================
+# SCRAPE DATA FROM ESTAR SITE AND GENERATE STOPPING POWER HDF5 FILE
+
+print('Generating stopping_powers.h5...')
+
+with h5py.File('stopping_powers.h5', 'w') as f:
+
+    # Write energies
+    f.create_dataset('energy', data=energies)
+
+    for Z in range(1, 99):
+        print('Processing {} data...'.format(ATOMIC_SYMBOL[Z]))
+
+        # Update form-encoded data to send in POST request for this element
+        data['matno'] = '{:03}'.format(Z)
+        payload = urlencode(data).encode("utf-8")
+
+        # Retrieve data from ESTAR site
+        r = urlopen(url=base_url, data=payload).read()
```

Comment:
As other reviewers have pointed out, best practices advocate calling `urlopen` within a context manager to automatically clean up connection resources:
https://docs.python.org/3.6/howto/urllib2.html